ccm_web/client/src/index.tsx

```diff
@@ -2,17 +2,21 @@ import { SnackbarProvider } from 'notistack'
 import React from 'react'
 import ReactDOM from 'react-dom'
 import { BrowserRouter } from 'react-router-dom'
+import { ThemeProvider } from '@material-ui/core'
 
+import ccmTheme from './theme'
 import App from './App'
 import './index.css'
 
 ReactDOM.render(
   <React.StrictMode>
+   <ThemeProvider theme={ccmTheme}>
     <SnackbarProvider maxSnack={3}>
       <BrowserRouter>
         <App />
       </BrowserRouter>
     </SnackbarProvider>
+    </ThemeProvider>
```

Comment:
Looks like the `</ThemeProvider>` closing tag is indented once too far. Unfortunately this doesn't show up on linting, not sure why yet, some JSX rules I think aren't working.

Reply:
I couldn't catch that. Hopefully I fixed it now :)